README.md

```diff
@@ -65,7 +65,7 @@ just run the following command to get an environment that will run on either the
    docker run -it -p 8000:8000 --rm allennlp/allennlp:v0.6.1
    ```
 
-You can now test your installation with `./scripts/verify.py`.
+You can now test your installation with `test-install`.
```

Comment:
`s/test-install/allennlp test-install/`?

Reply:
In docker you `docker run -it -p 8000:8000 --rm allennlp/allennlp:v0.6.1 test-install` rather than `docker run -it -p 8000:8000 --rm allennlp/allennlp:v0.6.1 allennlp test-install` (which won't work).

This is a bit confusing however.  Do you think I should just include the entire docker command?

Reply:
ah right, sorry --- yeah, I think the entire docker command would be better here.